plasmapy/physics/dispersion/two_fluid.py

```diff
@@ -0,0 +1,127 @@
+# FUNCTION TO CALCULATE PHASE SPEEDS OF THE THREE BRANCHES OF TWO FLUID
+# DISPERSION RELATION (e.g. STRINGER JPP 1963, ROGERS PRL 2001,
+# and BELLAN JGR 2012)
+
+#
+#                   Tulasi Nandan Parashar
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def tfps(beta=0.6, speed=1., de2=0.000545, theta=0, wavenumber=1):
+    """
+    beta: Total plasma beta,
+    speed: Alfven speed based on mean field,
+    de2: me/mi,
+    theta: Angle of propagation in degrees
```

Comment:
I think this is the angle between the propagation direction of the MHD wave and the magnetic field.